GEOSldas_App/ldas_setup

```diff
@@ -1238,17 +1243,23 @@ class LDASsetup:
         for nmlfile in default_nml:
             shortfile=self.rundir+'/'+nmlfile.split('/')[-1]
             shutil.copy2(nmlfile, shortfile)
-         # special nml
+        # special nml
         special_nml=[]
-        if 'NML_INPUT_PATH' in self.rqdExeInp :
-            special_nml = glob.glob(self.rqdExeInp['NML_INPUT_PATH']+'/LDASsa_SPECIAL_inputs_*.nml')
-            if self.ladas_cpl > 0: 
-               special_nml = glob.glob(self.rqdExeInp['NML_INPUT_PATH']+'/'+ self.rqdExeInp['EXP_DOMAIN'] + '/LDASsa_SPECIAL_inputs_*.nml') 
-  
-        for nmlfile in special_nml:
-            shortfile=nmlfile.split('/')[-1]
-            shutil.copy2(nmlfile, self.rundir+'/'+shortfile)
+        if self.ladas_cpl > 0:
+            special_nml= glob.glob(etcdir+'/LDASsa_SPECIAL_inputs_*.nml')
+        else : 
+            if 'NML_INPUT_PATH' in self.rqdExeInp : 
+                special_nml = glob.glob(self.rqdExeInp['NML_INPUT_PATH']+'/LDASsa_SPECIAL_inputs_*.nml')
```

Comment:
Thanks for cleaning up my messy handling of the special nml files.  This is much better!